soda/cmd/validate.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"github.com/petar/pop/soda/cmd/validate"
```

Comment:
You have imports that reference your fork. This might help you https://github.com/gobuffalo/buffalo/blob/master/CONTRIBUTING.md#first-time-committing-to-a-go-repo these imports need to be fixed.